src/library/Uno.Material/Styles/Controls/CheckBox.xaml

```diff
@@ -47,42 +47,11 @@
 							<VisualStateGroup x:Name="CombinedStates">
 
 								<VisualState x:Name="UncheckedNormal">
-									<xamarin:Storyboard>
-										<DoubleAnimation Storyboard.TargetName="PressRingTransform"
-														 Storyboard.TargetProperty="ScaleX"
-														 Duration="0:0:0.4"
-														 EasingFunction="{StaticResource Material_EaseInOutFunction}"
-														 From="0"
-														 To="1" />
 
-										<DoubleAnimation Storyboard.TargetName="PressRingTransform"
-														 Storyboard.TargetProperty="ScaleY"
-														 Duration="0:0:0.4"
-														 EasingFunction="{StaticResource Material_EaseInOutFunction}"
-														 From="0"
-														 To="1" />
-										
-										<android:DoubleAnimation Storyboard.TargetName="PressRing"
-																 Storyboard.TargetProperty="Opacity"
-																 Duration="0:0:0.5"
-																 EasingFunction="{StaticResource Material_EaseInOutFunction}"
-																 From="1"
-																 To="0" />
-
-										<DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="(FrameworkElement.Opacity)"
-																	   Storyboard.TargetName="PressRing">
-											<EasingDoubleKeyFrame KeyTime="0"
-																  Value="0.5" />
-											<EasingDoubleKeyFrame KeyTime="0:0:0.2"
-																  Value="0.5" />
-											<EasingDoubleKeyFrame KeyTime="0:0:0.4"
-																  Value="0" />
-										</DoubleAnimationUsingKeyFrames>
-									</xamarin:Storyboard>
 								</VisualState>
 
 								<VisualState x:Name="UncheckedPointerOver">
-									<Storyboard>
+									<!--<Storyboard>
```

Comment:
I'd try to avoid commented xaml with no explanation. Can we either remove the code or explain why it's commented?

Reply:
will remove